pageserver/src/tenant.rs

```diff
@@ -1614,6 +1624,27 @@ impl Tenant {
             Err(anyhow::anyhow!("failpoint: timeline-delete-after-rm"))?
         });
 
+        let remote_client = match &timeline.remote_client {
+            Some(remote_client) => remote_client,
+            None => return Ok(()),
+        };
+
+        remote_client.delete_all().await?;
+
+        // Have a failpoint that can use the `pause` failpoint action.
+        // We don't want to block the executor thread, hence, spawn_blocking + await.
+        #[cfg(feature = "testing")]
```

Comment:
I would prefer

```suggestion
        if cfg!(feature = "testing") {
```

here, so that (1) this code will be checked by clippy (2) avoid potential unused imports warnings if something is only used in this part

Reply:
Thanks! Pushed, changed it in both places